Resources/Textures/_Goobstation/Clothing/Head/Hardsuits/dreadnought.rsi/meta.json

```
@@ -1,7 +1,8 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Sprited by @b11jj for Goobstation, vulps sprites edited by YERO",
+  "copyright": "Taken from Paradise SS13 at commit https://github.com/ParadiseSS13/Paradise/commit/a67c929b7394f78e7787114457ba42f4df6cc3a1. Vox state by Flareguy for SS14, Resprited by Sunrise. monkey state made by MercerBray for Ss14",
```

Biome (2.4.15)

[error] The key copyright was already declared.

(lint/suspicious/noDuplicateObjectKeys)